### ISSUE TITLE
Final Peer Review - ams792

### ISSUE THREAD

**Issue by adams583** [akumar2017/NBA-Rookie-Predictions#17]
This project's goal was to predict the performance of NBA athletes based on their rookie year stats. This group used data from basketball-reference.com and NBA2K data as a dataset. The NBA2K data was used as a proxy for NBA performance, or how good a player is at a certain time. Overall I really liked this project, great work.  

Three things I like about this report:
- I thought it was creative and accurate to use NBA2K data as labels. 
- Great job feature engineering, preprocessing your data, and taking into account the effect of outliers in the analysis of your models
- Good variety in the models you used. Additionally, the explanations of each model, their drawbacks, and how you adjusted their hyperparameters were useful and clear

Three things I think could be improved on: 
- Possibly including more features, maybe some time-based to take into account a player's improvement over a year or so could be useful in predicting their career performance
- It would've been nice to have a little more EDA included in the report - I was curious what some of the features meant and which you thought might be most significant
- It would've been cool to try out different loss functions given that your data had a lot of outliers, i..e it seemed like you guys knew a lot about why your models weren't performing as well as you wanted, so choosing different loss functions to try to combat this might have improved something
